plugins/update-po-files.js

```diff
@@ -0,0 +1,49 @@
+module.exports = function (grunt) {
+
+    'use strict';
+
+    const execSync = require('child_process').execSync;
+
+    const task = function () {
+        const msgmergeLookup = 'which msgmerge;echo $?';
+        const msgmergePresent = !parseInt(execSync(msgmergeLookup, 'utf8'), 10);
+
+        if (!msgmergePresent) {
+            grunt.fatal(
+                'msgmerge is not available on the system. Please make sure gettext is' +
+                ' installed. You may have to do \'brew link gettext --force\' if on OS X.'
+            );
+        }
+
+        this.options({
+            template: 'template.pot'
+        });
+
+        const templateFile = this.options().template;
+
+        if (!this.filesSrc.length) {
```

Comment:
I thought the computed file array ppeared as `this.files.src`. Did they update that or is `this.fileSrc` something else?